### PR TITLE
Test case for #2764

### PR DESCRIPTION
**Pull request by bulkan**
Sample testcase for https://github.com/sequelize/sequelize/issues/2764

### CODE REVIEW

**Comment by mickhansen** [comment]
What's the significance of `ARRAY(JSON)` vs just `JSON` which supports arrays?

**Comment by bulkan** [comment]
Honestly I don't know why I went with `ARRAY(JSON)` instead of `JSON`. After a quick discussion I'm switching to a `JSON`

**Comment by mickhansen** [comment]
@bulkan i'm fixing it in any case, might aswell ;p it's not too tough a fix.

File: test/dialects/postgres/dao.test.js [hunk 1]
```diff
@@ -34,11 +38,34 @@ if (dialect.match(/^postgres/)) {
 
     it('should be able to search within an array', function(done) {
       this.User.all({where: {email: ['hello', 'world']}}).on('sql', function(sql) {
-        expect(sql).to.equal('SELECT "id", "username", "email", "settings", "document", "phones", "emergency_contact", "createdAt", "updatedAt" FROM "Users" AS "User" WHERE "User"."email" = ARRAY[\'hello\',\'world\']::TEXT[];');
+        expect(sql).to.equal('SELECT "id", "username", "email", "settings", "document", "phones", "emergency_contact", "friends", "createdAt", "updatedAt" FROM "Users" AS "User" WHERE "User"."email" = ARRAY[\'hello\',\'world\']::TEXT[];');
         done();
       });
     });
 
+    it('should be able update a field with type ARRAY(JSON)', function(){
```
Comment (fixe) [inline test/dialects/postgres/dao.test.js]: Typo: missing `to`.

Comment (mickhansen) [inline test/dialects/postgres/dao.test.js]: Thanks, fixing in my merge

**Comment by bulkan** [comment]
Thanks @mickhansen. Much appreciated.

**Pull request merged**